_extras/18-nextsteps.md

```diff
@@ -0,0 +1,117 @@
+>
+> - Your computational workflow and where advanced computing could help
+> - How to get access to facilities for your work
+> - How to get help and support to get your work running using advanced computing.
+>   For example, software development, further training, access to local expertise
+{: .callout}
+
+## Options for this session
+
+There are a number of different options for practical work during this session. The
+challenges below include: exploring your own work; an extended example using a parallel
+HPC application; an extended example using high throughput computing on multiple
+serial analyses. If you have something else you want to use the session for (e.g. to
+discuss things with the instructors/helpers as described above) then please feel free
+to do this. The idea of the session is to help you bootstrap your use of advanced computing
+and this will differ from individual to individual!
+
+> ## Exploring your work using HPC
+>
+> If you have a practical example of something from your area of work that you would like
+> help with getting up and running on an HPC system or exploring the performance of
+> on an HPC system, this is great! Please feel free to discuss this with us and ask
+> questions (both technical and non-technical).
+{: .challenge}
+
+> ## Exploring the performance of GROMACS
+>
+> [GROMACS](http://www.gromacs.org) is a world-leading biomolecular modelling package
+> that is heavily used on HPC systems around the world. Choosing the best resources
+> for GROMACS calculations is non-trivial as it depends on may factors, including:
+>
+> - The underlying hardware of the HPC system being used
+> - The actual system being modelled by the GROMACS package
+> - The balance of processes to threads used for the parallel calculation
+>
+> In this exercise, you should try and decide on a good choice of resources and settings
+> on {{ site.host }} for a typical biomolecular system. This will involve:
+>
+> - Downloading the input file for GROMACS from [{{ site.url }}{{site.baseurl }}/files/ion-channel.tpr]({{ site.url }}{{site.baseurl }}/files/ion-channel.tpr)
+> - Writing a job submission script to run GROMACS on {{ site.host }} using the system documentation
+> - Varying the number of nodes (from 1 to 32 nodes is a good starting point) used for the GROMACS job
+>   and benchmarking the performance (in ns/day)
+> - Using the results from this study to propose a good resource choice for this GROMACS calculation
+>
+> If you want to explore further than this initial task then there are a number of 
+> different interesting ways to do this. For example:
+> 
+> - Vary the number of threads used per process
+> - Reduce the number of cores used per node
+> - Allow the calculation to use Symmetric Mutithreading (SMT) if enabled
+>
+> Please ask for more information on these options from a helper!
+{: .challenge}
+
+> ## Running many serial BLAST+ analyses in parallel
```

Comment:
This is a very life-science based exercise and it is very involved. I think this is worth a capstone project. I would call it as such. Moreover, for people that have no professional relationship with Blast+, the outcome of the exercise might be irrelevant and thus demotivating. If the value of the outcome would be stressed more, I'll say I like the conjunction with gnu-parallel.

Reply:
Ok, I didn't understand it that way. if it is in extras and introduced as such, I am super happy to make it part of this repo.